src/main/java/servlets/YouTubeServlet.java

```diff
@@ -49,6 +53,18 @@ public static YouTube getService() throws GeneralSecurityException, IOException
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String DEVELOPER_KEY = "";
```

Comment:
Something to consider in the next pull request:

Instead of creating a scanner and reading the key every time doGet() is called, we could read it in a variable in an init function (kind of like a constructor), and then reuse it everytime.

https://stackoverflow.com/questions/1276082/how-to-initialize-a-servlet-during-startup-with-parameters